packages/components/src/internal/components/lineage/models.ts

```diff
@@ -99,6 +99,16 @@ export interface ProvenanceMap {
     to: Experiment.LineageItemBase;
 }
 
+export interface RunStepProperties {
```

Comment:
does this `RunStepProperties` interface need to be exported? I don't see any usages outside of this file